contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
that's an odd restriction. people other than apple package cross platform projects this way? or do they make a bunch of .dylib symlinks with the conventional name general code would expect to use? getting rid of this boolean in favor of more general solutions seems better to me.